### PR TITLE
Serverspec gen ui

### PR DESCRIPTION
**Pull request by Joeper214**
### UI enhancement for serverspec gen UI.

I have a few suggestions on the UI. 

- [x] Ace editor on (readonly mode).  and specify what formatting it is. ex. (.json, .js, .rb)
      ace

- [x] I think the accordion and close button are too close. maybe add a litte bit space.
      tooclose

### CODE REVIEW

**Comment by pocke** [comment]
Thanks, I'll review now

File: frontend/src/serverspec-gen/it-panel.ts [hunk 1]
```diff
@@ -20,6 +20,13 @@ export default Vue.extend({
     accordionToggle: function () {
       const el = (<HTMLElement>this.$el).querySelector('.collapse');
       $(el).collapse('toggle');
+
+      const up = (<HTMLElement>this.$el).querySelector('.glyphicon-chevron-up');
+      const down = (<HTMLElement>this.$el).querySelector('.glyphicon-chevron-down');
+      $(up).removeClass("glyphicon-chevron-up");
+      $(down).removeClass("glyphicon-chevron-down");
+      $(up).addClass("glyphicon-chevron-down");
+      $(down).addClass("glyphicon-chevron-up");
```
Comment (pocke) [inline frontend/src/serverspec-gen/it-panel.ts]: :+1:

Comment (Joeper214) [inline frontend/src/serverspec-gen/it-panel.ts]: :ok_hand:

**Comment by pocke** [comment]
Thanks!

**Pull request merged**